snuba/state/rate_limit.py

```diff
@@ -208,7 +208,7 @@ def get_global_rate_limit_params() -> RateLimitParameters:
     )
 
 
-class RateLimitAggregator(AbstractContextManager):
+class RateLimitAggregator(AbstractContextManager):  # type: ignore
```

Comment:
Shouldn't this be:
`class RateLimitAggregator(AbstractContextManager[RateLimitParameters]):`

Reply:
Python says that class is not Subscriptable.